Mathlib/Computability/AkraBazzi/AkraBazzi.lean

```diff
@@ -466,9 +466,9 @@ lemma base_nonempty {n : ℕ} (hn : 0 < n) : (Finset.Ico (⌊b (min_bi b) / 2 *
   have hb_pos : 0 < b' := R.b_pos _
   simp_rw [Finset.nonempty_Ico]
   exact_mod_cast calc ⌊b' / 2 * n⌋₊ ≤ b' / 2 * n := by exact Nat.floor_le (by positivity)
-                                 _ < 1 / 2 * n   := by gcongr; exact R.b_lt_one (min_bi b)
-                                 _ ≤ 1 * n       := by gcongr; norm_num
-                                 _ = n           := by simp
+                                 _ < 1 / 2 * n := by gcongr; exact R.b_lt_one (min_bi b)
+                                 _ ≤ 1 * n := by gcongr; norm_num
+                                 _ = n := by simp
```

Comment:
Is this "line up all the :="s style not allowed by the style guide? I think it adds something to readability; I would propose instead making the < on line 469 directly under the <= on 468 and leaving the := aligned. What do you think? The proposed change seems to be the worst of all worlds because nothing is lined up at all!

Reply:
I agree that having aligned := can add readability. At the same time, mathlib doesn't use that style consistently at all, and it's really easy for this alignment to get out of sync over time (and then it looks worse). For these reasons, I mostly prefer not having such alignment.

That said, this particular change is not great; let me try to find a better way.

Reply:
Pushed one.